src/excmds.ts

```diff
@@ -3316,20 +3316,16 @@ export function autocmd(event: string, url: string, ...excmd: string[]) {
     config.set("autocmds", event, url, excmd.join(" "))
 }
 
-/** Automatically open a domain and all its subdomains in a specified container.
+/** Automatically open matching URLs in a specified container.
  *
  *  For declaring containers that do not yet exist, consider using `auconscreatecontainer true` in your tridactylrc.
  *  This allows tridactyl to automatically create containers from your autocontain directives. Note that they will be random icons and colors.
  *
  * ** NB: This is an experimental feature, if you encounter issues please create an issue on github. **
  *
- *  The domain is passed through as a regular expression so there are a few gotchas to be aware of:
```

Comment:
It'd be handy if you put some examples here, e.g. one showing how to preserve the previous behaviour.

Reply:
Brought back the existing example, though it includes `google\,(co\.uk|com)` ... note the comma, I assume that's a typo?

Reply:
Yeah, should be a . :)